packages/Symfony/src/Rector/Console/ConsoleExecuteReturnIntRector.php

```diff
@@ -84,42 +99,119 @@ public function refactor(Node $node): ?Node
         return $this->addReturn0ToMethod($node);
     }
 
-    private function setReturnTo0InsteadOfNull(Return_ $return)
+    public function beforeTraverse(array $nodes): void
```

Comment:
This method should never be used in Rector rules as it is rather internal part (I know it's allowed, my bad, I gotta make it `final`).

What do you need to achieve here?

Reply:
as already said not necessary anymore ;)

Reply:
Understood :)